py_v_sdk/contract/tok_ctrt.py

```diff
@@ -0,0 +1,585 @@
+                attachment=md.Str(attachment),
+                fee=md.ExecCtrtFee(fee),
+            )
+        )
+        logger.debug(data)
+        return data
+
+    async def destroy(
+        self,
+        by: acnt.Account,
+        amount: int,
+        attachment: str = "",
+        fee: int = md.ExecCtrtFee.DEFAULT,
+    ) -> Dict[str, Any]:
+        """
+        Destroy an amount of tokens by account who has the issuing right
+
+        Args:
+            by (acnt.Account): The action taker
+            amount (int): The amount of token to be destroyed
+            attachment (str, optional): The attachment of this action. Defaults to "".
+            fee (int, optional): Execution fee of this tx. Defaults to md.ExecCtrtFee.DEFAULT.
+
+        Returns:
+            Dict[str, Any]: The response returned by the Node API
+        """
+
+        data = await by._execute_contract(
+            tx.ExecCtrtFuncTxReq(
+                ctrt_id=self._ctrt_id,
+                func_id=self.FuncIdx.DESTROY,
+                data_stack=de.DataStack(
+                    de.Amount(md.Int(amount)),
+                ),
+                timestamp=md.VSYSTimestamp.now(),
+                attachment=md.Str(attachment),
+                fee=md.ExecCtrtFee(fee),
+            )
+        )
+        logger.debug(data)
+        return data
+
+    async def transfer(
+        self,
+        by: acnt.Account,
+        sender: str,
+        recipient: str,
+        amount: int,
+        attachment: str = "",
+        fee: int = md.ExecCtrtFee.DEFAULT,
+    ) -> Dict[str, Any]:
+        """
+        transfer tokens from sender to recipient
+
+        Args:
+            by (acnt.Account): The action taker
+            sender (str): The sender account
+            recipient (str): The recipient account
+            amount (int): The amount to transfer
+            attachment (str, optional): The attachment of this action. Defaults to "".
+            fee (int, optional): Execution fee of this tx. Defaults to md.ExecCtrtFee.DEFAULT.
+
+        Returns:
+            Dict[str, Any]: The response returned by the Node API
+        """
+
+        sender_md = md.Addr(sender)
+        rcpt_md = md.Addr(recipient)
+
+        sender_md.must_on(by.chain)
+        rcpt_md.must_on(by.chain)
+
+        data = await by._execute_contract(
+            tx.ExecCtrtFuncTxReq(
+                ctrt_id=self._ctrt_id,
+                func_id=self.FuncIdx.TRANSFER,
+                data_stack=de.DataStack(
+                    de.Addr(sender_md),
+                    de.Addr(rcpt_md),
+                    de.Amount(md.Int(amount)),
+                ),
+                timestamp=md.VSYSTimestamp.now(),
+                attachment=md.Str(attachment),
+                fee=md.ExecCtrtFee(fee),
+            )
+        )
+        logger.debug(data)
+        return data
+
+    async def deposit(
+        self,
+        by: acnt.Account,
+        contract: str,
+        amount: int,
```

Comment:
The `amount` here can be `int | float`

See
https://github.com/virtualeconomy/py-v-sdk/blob/develop/py_v_sdk/account.py#L152
https://github.com/virtualeconomy/py-v-sdk/blob/develop/py_v_sdk/account.py#L174

The same applies to every method that requires `amount`